dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskConstants.java

```diff
@@ -362,24 +362,10 @@ private TaskConstants() {
      */
     public static final String DATA_QUALITY_JAR_DIR = "data-quality.jar.dir";
 
-    public static final String TASK_TYPE_CONDITIONS = "CONDITIONS";
-
-    public static final String TASK_TYPE_SWITCH = "SWITCH";
-
-    public static final String TASK_TYPE_SUB_PROCESS = "SUB_PROCESS";
-
-    public static final String TASK_TYPE_DYNAMIC = "DYNAMIC";
-
-    public static final String TASK_TYPE_DEPENDENT = "DEPENDENT";
-
-    public static final String TASK_TYPE_SQL = "SQL";
```

Comment:
Why remove this? I think it should be kept.

Reply:
This has been moved into `xxTaskChannelFactory`, this should be controlled by task plugin.